atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/TracingPrefsConfig.java

```diff
@@ -61,6 +62,8 @@ public void run() {
                             tracingPrefConfig.getProperty("min_duration_to_log_ms", "0"));
                     String tableString = tracingPrefConfig.getProperty("tables_to_trace", "");
                     tracedTables = ImmutableSet.copyOf(Splitter.on(",").trimResults().split(tableString));
+                    retryOnDifferentHostOnThriftTimedOutExceptions = Boolean.parseBoolean(
+                            tracingPrefConfig.getProperty("retryOnDifferentHostOnThriftTimedOutExceptions", "false"));
```

Comment:
seems odd that tracing prefs controls whether thrift timeouts retry. shouldn't this be part of the C* KVS config?